webpack.config.js

```diff
@@ -48,7 +48,6 @@ const gutenbergPackages = Object.keys( dependencies )
 const externals = {
 	react: 'React',
 	'react-dom': 'ReactDOM',
-	tinymce: 'tinymce',
```

Comment:
Don't we use this in the classic editor?

Reply:
No, it directly uses the `window` global.

Reply:
Ok that's fine for me.